eng/pipelines/common/upload-intermediate-artifacts-step.yml

```diff
@@ -1,9 +1,11 @@
 parameters:
   name: ''
+  publishPackagesCondition: always()
 
 steps:
 - task: CopyFiles@2
   displayName: Prepare job-specific intermediate artifacts subdirectory
+  publishPackagesCondition: and(succeeded(), ${{ parameters.publishPackagesCondition }})
```

Comment:
Hmm I'm not really good in yaml but is this a parameter to the copy files task?

Reply:
Shouldn't this just be `condition`?

Reply:
Good catch Viktor. I completely missed it.